.gitignore

```diff
@@ -1,4 +1,9 @@
+tester_*.py
```

Comment:
Has this file been upload by mistake?

Reply:
Thanks for catching this. I've recovered `.gitignore` as it was.